src/Docs/Resources/current/3-api/070-sales-channel-product-api.md

```diff
@@ -20,5 +20,11 @@ Description: Returns detailed information about a specific product.
 
 **GET /sales-channel-api/v1/product/{productId}?associations[media][]**
 
+You can also load deep associations by providing multiple associations keys.
+
+**GET /sales-channel-api/v1/product/{productId}?associations[categories][associations][media][]**
+
+This will load the category assoassociation and the meda association of the category.
```

Comment:
`assoassociation` => `association` 🤔

Reply:
the PR was already imported. we will fix that internally

Reply:
Sorry, was too fast on this one. Thx!